tests/func/clean/all.yml

```diff
@@ -9,3 +9,15 @@
     apply:
       tags:
         - test_subnet
+
+- include_tasks:
+    file: security_group.yml
+    apply:
+      tags:
+        - test_security_group
+
+- include_tasks:
+    file: security_group_rule.yml
+    apply:
+      tags:
+        - test_security_group_rule
```

Comment:
I would expect these in inverse order, i.e. first clean up rules then groups, but if it works either way then i guess it's ok. Maybe that will change if we fix the security group rule below.

Reply:
done